Frends.AmazonS3.DownloadObject/Frends.AmazonS3.DownloadObject/Definitions/SingleResultObject.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Frends.AmazonS3.DownloadObject.Definitions;
+
+/// <summary>
+/// Single download result.
+/// </summary>
+public class SingleResultObject
+{
+    /// <summary>
+    /// Single download result.
+    /// </summary>
+    /// <example>Download complete: {fullPath}.</example>
+    public string ObjectData { get; private set; }
+
+    /// <summary>
+    /// Object's name.
+    /// </summary>
+    /// <example>Filename.txt</example>
+    public string ObjectName { get; private set; }
+
+    /// <summary>
+    /// Full path.
+    /// </summary>
+    /// <example>c:\temp, \\network\folder</example>
+    public string FullPath { get; private set; }
+
+    internal SingleResultObject(string objectData, string objectName, string destinationDirectory)
```

Comment:
Maybe change `destinationDirectory` to `fullpath` to make it clear that this is a full path, not just path to directory.